packages/domain/réseau/src/raccordement/transmettre/transmettreDateMiseEnService.behavior.ts

```diff
@@ -43,6 +43,10 @@ export async function transmettreDateMiseEnService(
     throw new DossierRaccordementNonRéférencéError();
   }
 
+  if (!this.dateModifiée(référenceDossier, dateMiseEnService)) {
+    throw new DateMiseEnServiceDéjàTransmiseError();
```

Comment:
Petite suggestion (car j'ai mis du temps à comprendre la connexion entre l'erreur et la condition)

```suggestion
    throw new DateIdentiqueDeMiseEnServiceDéjàTransmiseError();
```